### PR TITLE
Add Dockerfiles for Alpine 3.19

### LINKED ISSUE #5052: Question/Request: Alpine 3.19
### Plans to add/update Alpine 3.19

Hello,
Last week Alpine 3.19 was released, bringing with a version bump to a fresh kernel and Raspberry Pi 5 support.
I am very interested in this, and would therefore like to ask if you already have this update on your backlog/roadmap (and I was just unable to find it), or if I could request that update as a "feature request"?

### Describe the Solution

Ideally, I would like to use "official" base images, as released from this project.
In the mean time, I can just bake my own to get started. This is not a blocking issue at all.

### Additional Context

Link to announcement: https://www.alpinelinux.org/posts/Alpine-3.19.0-released.html

### PR DESCRIPTION
**Pull request by lbussell**
Fixes #5052 

Todo:
- [ ] Update with real image sizes

### CODE REVIEW

**Comment by lbussell** [comment]
Filed https://github.com/PowerShell/PowerShell/issues/20945 for PowerShell issue.

File: src/aspnet/8.0/alpine3.18/alpine3.18-composite/amd64/Dockerfile
Comment (MichaelSimons) [inline src/aspnet/8.0/alpine3.18/alpine3.18-composite/amd64/Dockerfile:1]: How come these src/aspnet/8.0/alpine3.18/alpine3.18-composite Dockerfiles are being added in this PR?

Comment (lbussell) [reply src/aspnet/8.0/alpine3.18/alpine3.18-composite/amd64/Dockerfile:1]: Good catch, I think I made a mistake copying files around.

Comment (MichaelSimons) [reply src/aspnet/8.0/alpine3.18/alpine3.18-composite/amd64/Dockerfile:1]: Oh I see these are being added as a nested folder under the alpine3.18 folder when they already exist as a peer folder today -  https://github.com/dotnet/dotnet-docker/tree/nightly/src/aspnet/8.0/alpine3.18-composite

**Review by MichaelSimons — APPROVED** [review]

**Comment by mthalman** [comment]
> Filed https://github.com/PowerShell/PowerShell/issues/20945 for PowerShell issue.

The underlying issue for PowerShell is due to there being no RID defined: https://github.com/PowerShell/PowerShell/issues/16532#issuecomment-1015936946

**Comment by lbussell** [comment]
> > Filed [PowerShell/PowerShell#20945](https://github.com/PowerShell/PowerShell/issues/20945) for PowerShell issue.
> 
> The underlying issue for PowerShell is due to there being no RID defined: [PowerShell/PowerShell#16532 (comment)](https://github.com/PowerShell/PowerShell/issues/16532#issuecomment-1015936946)

I also filed https://github.com/dotnet/runtime/issues/96155 and https://github.com/dotnet/core/issues/9013.

**Review by mthalman — APPROVED** [review]
Due to the PowerShell issue we should avoid moving the floating `alpine` tag to 3.19 until it gets fixed. I know that's not being done in this PR but wanted to point it out (can probably be noted in the release tracking issue).

**Pull request merged**